app/celer-sim/Runner.hh

```diff
@@ -84,9 +86,11 @@ class Runner
 
     //// DATA ////
 
+    ImportData imported_;
```

Comment:
The imported data comes from the lambda function in `build_core_params`, which sets up what Importer type we use. Since we don't want to initialize Geant4 more than once, maybe I should set up the importer in the constructor and pass the world volume and import data as needed to the `build_[x]` helper functions?

Reply:
What if we save the world volume as class member data, do the import in the constructor, and pass the import data to the various parts that need it?

Reply:
That was one of my first thoughts actually, when dealing with passing the imported data from `build_core_params` to `build_optical_data`. I would either need a direct G4 include in our `Runner.hh` or have something like the `UPExtern` we use for ROOT. Which led me to add the data itself, despite being large. Do you think we should go for the unique ptr?

Reply:
No, since it's a raw pointer, you just have to forward declare:
```c++
class G4VPhysicalVolume;  // IWYU pragma: keep
```

Reply:
Oh, if you're okay with the raw pointer, then yes, that's easy. Come to think of it, the runner's lifetime is basically the celer-sim one, but the Geant4 application should end after we're done with the importing (currently in `build_core_params`). Wouldn't  this poses a problem with our world raw pointer (i.e. Geant4 deleting the geometry when it goes out of scope)?